src/coreclr/ilasm/asmparse.h

```diff
@@ -197,8 +187,7 @@ class MappedFileStream : public ReadStream {
         return (m_hFile == INVALID_HANDLE_VALUE) ? NULL : map_file();
     }
 
-    const WCHAR* fileNameW;     // FileName (for error reporting)
-	//char	fileNameANSI[MAX_FILENAME_LENGTH*4];
+	char	fileNameUtf8[MAX_FILENAME_LENGTH*4]; // FileName (for error reporting)
```

Comment:
Seeing a fixed buffer for file path always makes me feel uneasy. It seems that we could allocate it dynamically here without much hassle. Would you mind doing it that way?

Reply:
Sure.